src/OpenLoco/src/GameCommands/Terraform/LowerRaiseLandMountain.cpp

```diff
@@ -287,16 +287,20 @@ namespace OpenLoco::GameCommands
         }
 
         // 0x00462F25
+        const int16_t minRadius = ((args.pointB.x - args.pointA.x) >> 5) - 1;
+        const int16_t maxRadius = minRadius + 64;
```

Comment:
This is a change from vanilla, facilitating larger mountain table sizes allowed by #2228.